src/SurfaceFluxes.jl

```diff
@@ -286,6 +293,44 @@ function compute_physical_scale(
     return _π_group⁻¹ * von_karman_const / Σterms * (x_in - x_s)
 end
 
+function compute_physical_scale_coeff(
```

Comment:
this corresponds to F_\phi and F_m in equations (7) and (8) in the docs?
if so how does this function knows if it's computing for momentum or scalars?